modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc

```diff
@@ -22,6 +31,7 @@ To successfully install ROSA clusters, use the latest version of the ROSA CLI.
 * You have verified that the AWS Elastic Load Balancing (ELB) service role exists in your AWS account.
 * You have associated your AWS account with your Red Hat organization. When you associated your account, you applied the administrative permissions to the {cluster-manager} role. For detailed steps, see _Associating your AWS account with your Red Hat organization_.
 * You have created the required account-wide STS roles and policies, including the Operator policies. For detailed steps, see _Creating the account-wide STS roles and policies_.
```

Comment:
remove:
', including the Operator policies'

Because we now only create them in the `rosa create operatorroles` step.